config/upgrade/v0.15.0/README.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade script (optional) to upgrade to v0.15.0 of Eventing
+
+This directory contains a job that updates all of v1alpha1 based Brokers that
+are using Spec.ChannelTemplate to specify which channel they use. You are not
+able to create new ones specifying the ChannelTemplate, and you can either
+manually update them yourself, or run the tool that will do the following for
+any Broker that is using Spec.ChannelTemplate:
+
+1. Create a ConfigMap in the same namespace as the Broker named:
```

Comment:
The tool creates a ConfigMap ? 

I first read it as user instructions - hence the comment for code snippet :-)